community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/LabelTokenRecord.java

```diff
@@ -32,4 +32,19 @@ protected String simpleName()
     {
         return "Label";
     }
-}
\ No newline at end of file
+
+    @Override
+    public LabelTokenRecord clone()
```

Comment:
No particular reason, it was that way for the other records, simply added the `clone()` to the new ones. We can cleanup later in another PR.